src/api/protectedApiClient.ts

```diff
@@ -29,15 +29,15 @@ const changePassword = (request: {
     .catch((e) => e);
 };
 
-const deleteUser = (request: { password: string }): Promise<void> => {
-  return AppAxiosInstance.post(ProtectedApiClientRoutes.DELETE_USER, request)
-    .then((r) => r.data)
+const decactivateAccount = (): Promise<void> => {
+  return AppAxiosInstance.delete(ProtectedApiClientRoutes.USER)
+    .then((r) => r)
```

Comment:
I think the `then` statement is useless if this function is returning `Promise<void>`

Reply:
I prefer `res` and `err` here, makes it more readable + grep-able IMO

Reply:
the `.then` should be `.then(() => {})` I think, as it's void. Error should be `.catch((err) => err)`

I think we need the .then to resolve from an AxiosPromise to a Promise?